run_chesapeake_cvpr_experiments.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Runs the train script with a grid of hyperparameters."""
+import itertools
+import os
+import subprocess
+from multiprocessing import Process, Queue
+
+# list of GPU IDs that we want to use, one job will be started for every ID in the list
+GPUS = [0, 1, 2]
+TEST_MODE = False  # if False then print out the commands to be run, if True then run
+
+# Hyperparameter options
+training_set_options = ["de"]
+# model_options = ["unet", "deeplabv3+"]
+model_options = ["fcn"]
+lr_options = [1e-2, 1e-3, 1e-4]
+loss_options = ["ce", "jaccard", "focal"]
+
+
+def do_work(work: "Queue[str]", gpu_idx: int) -> bool:
+    """Process for each ID in GPUS."""
+    while not work.empty():
+        experiment = work.get()
+        experiment = experiment.replace("GPU", str(gpu_idx))
+        print(experiment)
+        if not TEST_MODE:
+            subprocess.call(experiment.split(" "))
+    return True
+
+
+def main() -> None:
+    """Main."""
```

Comment:
This docstring isn't very descriptive...

Reply:
removed